settings.gradle.kts

```diff
@@ -49,6 +49,10 @@ val enableINIFrontend: Boolean by extra {
     val enableINIFrontend: String? by settings
     enableINIFrontend.toBoolean()
 }
+val enableConcepts: Boolean by extra {
```

Comment:
also remove everything related to concepts.

Reply:
done